<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLineEdit" name="pathFileEdit_Excel"/>
   </item>
   <item row="4" column="0">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="pathFileEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="openOutFileBk">
     <property name="text">
      <string>打開输出文件</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="openFileBt">
     <property name="text">
      <string>打開文件</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="pathFileEdit_outPath"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="openExcelFileBk">
     <property name="text">
      <string>打開excel翻译表</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>查找方式</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
